nightly.yml

```diff
@@ -10,10 +10,10 @@
       inputs:
           BuildDropPath: '$(Build.ArtifactStagingDirectory)\$(NugetArtifactsDir)\sbom'
 
-    - task: PublishBuildArtifacts@1
-      displayName: 'Publish SBOM'
-      inputs:
-        PathtoPublish: '$(Build.ArtifactStagingDirectory)\$(NugetArtifactsDir)\sbom'
+    # - task: 1ES.PublishPipelineArtifact@1
+    #   displayName: 'Publish SBOM'
+    #   inputs:
+    #     path: '$(Build.ArtifactStagingDirectory)\$(NugetArtifactsDir)\sbom'

```

Comment:
Do we not need this any more?
If we don't, shouldn't we get rid of the task instead of commenting it out?